### ISSUE TITLE
How do I submit my results?

### ISSUE THREAD

**Issue by Lvqin001** [Yuliang-Liu/Curve-Text-Detector#47]
Hello, can I submit my results to this list? Does it have to be accompanied by a paper explaining the method? I look forward to your answer very much.

**Comment by zhangshuaitao** [comment]
@Lvqin001 , We provide two metrics here, and you can simply convert your results into the sample format we released to test the results directly with our code.

**Comment by Yuliang-Liu** [comment]
@Lvqin001 Yes. We will update the results according to recent published paper. You can post the link here if I miss your paper.
